week4/MealsharingNet/Models/Meal.cs

```diff
@@ -0,0 +1,12 @@
+namespace MealsharingNet.models;
+
+public class Meal
+{
+    public int ID { get; set; }
+    public string Title { get; set; }
+    public string Description { get; set; }
+    public string Location { get; set; }
+    public int MaxReservation { get; set; }
+    public int Price { get; set; }
```

Comment:
The price can potentially be something like 9.99 so a more appropriate type would be decimal instead of int